logstash-core/lib/logstash/pipeline.rb

```diff
@@ -159,7 +160,8 @@ def run
     shutdown_flusher
     shutdown_workers
 
-    @logger.log("Pipeline #{@pipeline_id} has been shutdown")
```

Comment:
The reason for `log` (no level) here instead of `info` is because there are certain kinds of events in Logstash's life that has caused confusion or frustration for users. I understand log4j2 has no concept of a log with no "level", though. I guess the only equivalent is `info` (since it is not conceptually a warning/error).